interfaces/builtin/desktop_launch.go

```diff
@@ -48,7 +48,7 @@ dbus (send)
     bus=session
     path=/io/snapcraft/PrivilegedDesktopLauncher
     interface=io.snapcraft.PrivilegedDesktopLauncher
-    member=OpenDesktopEntry
+    peer=(label=unconfined),member=OpenDesktopEntry
```

Comment:
This looks wrong.  The comma separates AppArmor rules, so this breaks a rule in half.

Rather than changing the AppArmor snippet, @pedronis was suggesting that an extra assertion be added to the test verifying that the snippet contained this part of the rule.